content_scripts/link_hints.coffee

```diff
@@ -673,5 +674,25 @@ class TypingProtector extends Mode
 
     @onExit callback
 
+# TODO(mrmr1993): Use event.deepPath to traverse into shadow DOMs, as part of a solution to #1861. This
```

Comment:
Oh I see. Sorry for my carelessness and poor English - you've added a note in `hook_addEventListener.coffee:81`.